src/screens/Home/Home.tsx

```diff
@@ -1,30 +1,83 @@
 import * as React from 'react';
 import { LatestPostsQuery } from '../../generated/graphql';
 import { Link } from 'react-router-dom';
+import Row from 'react-bootstrap/Row';
```

Comment:
(nit) This needs to be ordered alphabetically by package name. (My bad eventually, it wasn't before)
also locally imported types / functions should be separated by a blank line.

so :
```js
import * as React from 'react';
import Col from 'react-bootstrap/Col';
import Row from 'react-bootstrap/Row';
import { Link } from 'react-router-dom';
import styled from 'styled-components';

import DiscussionCard from '../../components/DiscussionCard'
import { LatestPostsQuery } from '../../generated/graphql';
```